EIPS/eip-3450.md

```diff
@@ -0,0 +1,151 @@
+---
+eip: 3450
+title: Standardized Shamir Secret Sharing Scheme for BIP-39 Mnemonics
+author: Daniel Streit (@danielstreit)
+status: Draft
```

Comment:
```suggestion
discussions-to: <URL>
status: Draft
```
`discussions-to` link is required.  We recommend a thread on https://ethereum-magicians.org/ but if that is not possible or you are unwilling to create one there then an issue in this repository can be used as an alternative.

Reply:
Added.